v19.2/interval.md

```diff
@@ -8,7 +8,12 @@ The `INTERVAL` [data type](data-types.html) stores a value that represents a spa
 
 ## Aliases
 
-There are no aliases for the interval type. However, CockroachDB supports using uninterpreted [string literals](sql-constants.html#string-literals) in contexts where an `INTERVAL` value is otherwise expected.
+- `INTERVAL(6)`
```

Comment:
~The list formatting here is kind of weird since there's only one item, but it's probably best to leave it so we can add things easily in future.